src/Conversion/ONNXToMhlo/Math/MatMul.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- Matmul.cpp - Lowering Matmul Op --------------------===//
+//
+// Copyright 2019-2022 The IBM Research Authors.
```

Comment:
I only put the date this time. Because I don't know what process to go through to put the company name (ByteDance) here. It may take some time to figure out.

Reply:
Then just leave your company out... but it does not seems to be right to add `IBM Authors` when you are not employed by IBM, if you see what I mean.

Reply:
I didn't add `IBM Authros`, either. Only `Copyright 2022` currently.